webapp/src/components/ProgramCalendar.tsx

```diff
@@ -45,7 +45,18 @@ const ProgramCalendar = ({ program }: ProgramCalendarProps) => {
     end: new Date(),
     programTitle: '',
   });
-  const [currentView, setCurrentView] = React.useState<string>(Views.WEEK);
+  const [currentWidth, setCurrentWidth] = React.useState<number>(
+    window.innerWidth
+  );
+  const [currentView, setCurrentView] = React.useState<any>(
+    currentWidth <= 600 ? Views.DAY : Views.WEEK
```

Comment:
When I was trying to test the functionality in my local machine it gives me an error and display an empty page if I am in the week view and make my window smaller than 600px. If I refresh it shows the daily view again...
have you noticed that?

https://user-images.githubusercontent.com/98124157/206277543-615988d1-0794-46d7-bab4-da6c71eb4d3d.mov

Reply:
I noticed it when I resized the screen very fast. I'm not sure why it doesn't work. Anyway, I'm trying to refactor the code. Thank you for letting me know.

Reply:
This is happening on my end as well

Reply:
@rrabinovitch It's weird. I updated  code and it should be work now. Could you describe when it was happen?